discord/asset.py

```diff
@@ -106,6 +107,14 @@ def _from_guild_image(cls, state, id, hash, key, *, format='webp', size=1024):
 
         url = 'https://cdn.discordapp.com/{key}/{0}/{1}.{2}?size={3}'
         return cls(state, url.format(id, hash, format, size, key=key))
+    
+    @classmethod
+    def _from_widget_image(cls, state, id, *, style='shield'):
+        if format not in VALID_STATIC_FORMATS:
```

Comment:
You check if `format` is in `VALID_STATIC_FORMATS` but ``format`` doesn't exist...?

Reply:
Yeah, that was kind of dumb. I was working on these changes at school in the Github editor (because that is where I have the most time to work on this stuff) so that is where that kind of dumb error comes from. Thanks for the "spellcheck" so to speak.

Reply:
a8c929105594ce2c5d3889e4c0ae5abf464d12c3